Phindr3D-Python/src/GUI/windows/plot_functions.py

```diff
@@ -36,12 +36,13 @@
 def treatment_bounds(self, data, bounds, id):
 
     try:
-        trt = data['Treatment'].iloc[id]
-        trt_loc =int(np.where(data['Treatment'].unique() == trt)[0])
+        trt = data[self.metadata.treatmentColNameForNormalization].iloc[id]
```

Comment:
Should this always be treatmentColNameForNormalization? You can also use metadata.GetTreatmentColumnName()